modules/appeals_api/app/services/appeals_api/pdf_construction/supplemental_claim/v2/structure.rb

```diff
@@ -129,9 +144,10 @@ def fill_autosize_fields
             end
             tmp_path
           end
+          # rubocop:enable Metrics/MethodLength
 
           def additional_pages?
-            additional_issues? || additional_evidence_locations? || form_data.long_signature?
+            additional_issues? || additional_evidence_locations? || form_data.long_signature? || form_data.long_name?
```

Comment:
I can't seem to remember - did we decide that if a name is too long we add it to an additional page? Or should this be removed and just keep the check for the long signature?